program.cs

```diff
@@ -5,3 +5,5 @@
 int k = j / q;
 
 Console.Write(k);
+
+// Comment
```

Comment:
Why are you adding a comment when you don't need one.

Reply:
I will resolve this